metadata-ws/src/test/resources/application.properties

```diff
@@ -20,4 +20,6 @@ swagger.apiinfo.licenseUrl=http://www.apache.org/licenses/LICENSE-2.0
 swagger.apiinfo.version=@project.version@
 swagger.apiinfo.termsOfServiceUrl=
 
-endpoints.entities.release-date.restricted=true
\ No newline at end of file
+endpoints.entities.release-date.restricted=true
+
+security.enabled=false
```

Comment:
Could you please add documentation in the `docs/` subfolder for how the authentication works, which external server is used and who manages it?

Reply:
I document once i get input from. To track i have created a ticket https://www.ebi.ac.uk/panda/jira/browse/T2D-318